internal/elasticsearch/client.go

```diff
@@ -84,6 +87,16 @@ func DeleteIndex(ctx context.Context, index string) error {
 // random port at localhost, we will build the URL with the bound port at localhost.
 //nolint:unused
 func getElasticsearchClient(ctx context.Context) (*es.Client, error) {
+	remoteESHost := shell.GetEnv("ELASTICSEARCH_URL", "")
```

Comment:
Can we add this variable to the README doc where we explain the variables affecting the build? https://github.com/elastic/e2e-testing/tree/master/e2e#environment-variables-affecting-the-build

Reply:
I've just realised we are setting elasticsearch and kibana hosts for metricbeat in its compose file under `config/metricbeat`. Could you please update it too?